src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpHeaders.cs

```diff
@@ -942,6 +947,9 @@ private static void AddValue(HeaderStoreItemInfo info, object? value, StoreLocat
                     currentStoreValue = info.RawValue;
                     AddValueToStoreValue<string>(value, ref currentStoreValue);
                     info.RawValue = currentStoreValue;
+                    currentStoreValue = info.OriginalRawValue;
```

Comment:
It seems awkward and nonobvious to set both RawValue and OriginalRawValue here. Why not just add StoreLocation.OriginalRaw and make this explicit?

Reply:
My goal is to minimize changes to the existing code, so I tried updated as few places as possible. If I implemented a new `StoreLocation.OriginalRaw` I would have to add `AddValue(StoreLocation.OriginalRaw)` calls to all places where  `AddValue(StoreLocation.Raw)` is called.

Reply:
> I would have to add AddValue(StoreLocation.OriginalRaw) calls to all places where AddValue(StoreLocation.Raw) is called.

How many are there?  Three?